src/node_url.cc

```diff
@@ -313,7 +313,7 @@ void Initialize(Local<Object> target,
                 void* priv) {
   SetMethod(context, target, "parse", Parse);
   SetMethod(context, target, "updateUrl", UpdateUrl);
-  SetMethod(context, target, "formatUrl", FormatUrl);
+  SetMethodNoSideEffect(context, target, "formatUrl", FormatUrl);
 
```

Comment:
Maybe change the line break in between? It seems like they distinguished the regular from the no side effect ones before :)

Reply:
I mainly did that in my previous pull request to distinguish URL parser related functions with others, but I see the conclusion :-) Should I still do it?